### PR TITLE
Extract new strings (2023-04-03)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: en-US/firefox-ios.xliff
Comment (flodolo) [reply en-US/firefox-ios.xliff:1791]: How is the percentage formatted currently?

Internationalization libraries have [that information](https://unicode-org.github.io/cldr-staging/charts/latest/by_type/numbers.number_formatting_patterns.html#17acd127f9139476). If they're using one, and not just doing `X%` (replacing X with a number), then it would be OK.

Comment (Delphine) [reply en-US/firefox-ios.xliff:1791]: Thank you Flod.
CC @PARAIPAN9 and/or @q2r5 (I think one of you landed that string): could you help confirm the percentage format and if it's currently using an internationalization library? Just want to be extra cautious. thanks

Comment (flodolo) [reply en-US/firefox-ios.xliff:1791]: The alternative is to actually expose the percentage (`Zoom (%@%)`), but given that symbol is used for the variable placeholder, I expect lots of errors…

Comment (PARAIPAN9) [reply en-US/firefox-ios.xliff:1791]: Hello @flodolo, @Delphine, in code, we are using this format: "%.0f%%"

Comment (flodolo) [reply en-US/firefox-ios.xliff:1791]: Which is wrong for several locales (including French, German, and Spanish). See previous link for all locales using `#,##0 %`, with a space before the symbol.

Comment (Delphine) [reply en-US/firefox-ios.xliff:1791]: Hi @PARAIPAN9, gentle ping to see if you were able to update the string, and so we can expose them to localization. Thank you!

**Review by Delphine — CHANGES_REQUESTED** [review]

File: en-US/firefox-ios.xliff
Comment (Delphine) [reply en-US/firefox-ios.xliff:1791]: It seems like a fix was made for this, and I missed it being referenced in this PR...
I synced in with @bcolsson for this, since I'm still trying to understand the fix.
I'll merge later today unless something comes up. Thanks all

Comment (Delphine) [reply en-US/firefox-ios.xliff:1791]: I went through the swift code (and docs) and I understand very little, so I'll merge this and will monitor once the string lands that things work now as expected.

File: en-US/firefox-ios.xliff
Comment (Delphine) [inline en-US/firefox-ios.xliff:4353]: Nit: there are two placeholders here, not one
Please explain for both in comments

File: en-US/firefox-ios.xliff
Comment (Delphine) [inline en-US/firefox-ios.xliff:3243]: Comment should explain what placeholder will be replaced with.
Also, this string assumes that all languages use plural the same way as English (only two forms), which they do not.
Since we don't support proper plurals at the time, a workaround would be to format the string as "Tabs closed: %d" (just leave "tabs" as plural, even if it's not ideal)
No need to update string ID since strings haven't been exposed yet

**Review by Delphine — CHANGES_REQUESTED** [review]
More strings have landed since my last review, and I found a couple things to update.

**Review by Delphine — CHANGES_REQUESTED** [review]
Hey @adudenamedruby 
* "CloseTabsToast.Title.Plural.v113"
* "CloseTabsToast.Title.Singular.v113"
Sooo... sorry, this still doesn't work 😜 
The reason is that both strings still imply that there are only two plural rules for all languages: 1 is singular and the rest is plural.
As you can see [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms), this is not the case for all languages. The bottom section details the xliff case: "For some formats, plural forms are not supported, for a limitation in either the format itself (DTD, JSON) or in our toolchain (XLIFF, XML). In these cases, you should try to use plural neuter forms: instead of %S files were copied, use Copied files: %S."
So since we don't support proper plural forms, the workaround would be to have just one string that reads: "Tabs closed: %d"
thanks!

**Comment by adudenamedruby** [comment]
@Delphine I updated copy & behaviour, so we should be good when you update again. :)

**Review by Delphine — APPROVED** [review]

**Pull request merged**
